lib/renderPopup.ts

```diff
@@ -27,9 +27,10 @@ function renderLayer(layerId: string) {
 
 function renderProperties(feature: GeoJSONFeatureWithSourceLayer) {
```

Comment:
what is the return type of this function? A string?

Reply:
Yes, a html kind of string I think...